code/modules/cooking/machinery/cooking_machines/container.dm

```diff
@@ -148,13 +144,119 @@
 	shortname = "shelf"
 	desc = "Put ingredients in this; designed for use with an oven. Warranty void if used."
 	icon_state = "ovendish"
-	center_of_mass = list("x" = 16,"y" = 12)
 	max_space = 30
-	max_reagents = 120
-	unacidable = TRUE
+	volume = 120
+	appliancetype = OVEN
+
+/obj/item/reagent_containers/cooking_container/skillet
+	name = "skillet"
+	shortname = "skillet"
+	desc = "Chuck ingredients in this to fry something on the stove."
+	icon_state = "skillet"
+	volume = 15
+	force = 15
+	hitsound = 'sound/weapons/smash.ogg'
+	flags = OPENCONTAINER // Will still react
+	appliancetype = SKILLET
+
+/obj/item/reagent_containers/cooking_container/skillet/Initialize(var/mat_key)
+	. = ..()
+	var/material/material = SSmaterials.get_material_by_name(mat_key || MATERIAL_STEEL)
+	if(material.name != MATERIAL_STEEL)
+		color = material.icon_colour
+	name = "[material.display_name] [initial(name)]"
+
+/obj/item/reagent_containers/cooking_container/saucepan
+	name = "saucepan"
+	shortname = "saucepan"
+	desc = "Is it a pot? Is it a pan? It's a saucepan!"
+	icon_state = "pan"
+	volume = 60
+	slot_flags = SLOT_HEAD
+	force = 15
```

Comment:
Jesus fuck that's way too strong for a saucepan

Reply:
lmao it was originally 30, i asked for advice but no one i asked had any advice on what to set it to
```suggestion
	force = 5
```